deployment/CE04OSSM_Deploy.csv

```diff
@@ -492,32 +492,61 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
-18,CE04OSSM-SBD11-06-METBKA002,CGINS-METBPR-00257,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-4,575,,,B
-18,CE04OSSM-SBD11-06-METBKA003,CGINS-METHRH-00294,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-4,575,,,B
-18,CE04OSSM-SBD11-06-METBKA004,CGINS-METLWR-00250,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-4,575,,,B
-18,CE04OSSM-SBD11-06-METBKA005,CGINS-METPRC-00267,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-4,575,,,B
-18,CE04OSSM-SBD11-06-METBKA006,CGINS-METSWR-00251,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-4,575,,,B
-18,CE04OSSM-SBD11-06-METBKA007,CGINS-METWND-00248,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,-5,575,,,B
-18,CE04OSSM-SBD11-01-MOPAK0000,CGINS-MOPAK0-11856,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,0,575,,,B
-18,CE04OSSM-SBD12-04-PCO2AA000,CGINS-PCO2AA-35257,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,0.7,575,,,B
-18,CE04OSSM-SBD11-04-VELPTA000,CGINS-VELPTA-08535,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,1.3,575,,,B
-18,CE04OSSM-SBD12-05-WAVSSA000,CGINS-WAVSSA-06291,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,0,575,,,B
-18,CE04OSSM-RID26-01-ADCPTC000,CGINS-ADCPTC-22808,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID27-03-CTDBPC000,CGINS-CTDBPC-07241,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID27-04-DOSTAD000,CGINS-DOSTAD-00223,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID27-02-FLORTD000,CGINS-FLORTD-01488,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID26-07-NUTNRB000,CGINS-NUTNRB-01057,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID27-01-OPTAAD000,CGINS-OPTAAD-00174,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID26-06-PHSEND000,CGINS-PHSEND-P0123,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID26-08-SPKIRB000,CGINS-SPKIRB-00298,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
-18,CE04OSSM-RID26-04-VELPTA000,CGINS-VELPTA-11780,2024-10-16T18:28:00,,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,,,500,7,575,,,B
+18,CE04OSSM-SBC11-00-CPMENG000,CGCON-BCPM01-50007,2024-10-16T18:28:00,2025-05-27T21:06:00,44.37883,-124.94883,CGMCE-04OSSM-00018,SKQ202415S,,1,,SR2511,,500,0,575,,,B
```

Comment:
I don't see any specific documentation of 21:06 UTC / 14:06 PDT as the clip in or release time in the daily cruise log, however, the event times that are recorded bracket this time and 14:06 PDT checks out for an appropriate recovery clip time given the description of tasks leading up to recovery.
So no change required, just noting.